app/models/people/mentor.rb

```diff
@@ -38,7 +38,7 @@ def self.find_or_create_from_reg_id(reg_id)
   # Returns true if +passed_background_check?+ and +signed_risk_form?+ and +currently_enrolled?+ all return true, and
   # if has attended mentor workshop (if necessary)
   def passed_basics?
-    (!Customer.require_background_checks? || (passed_background_check? && passed_sex_offender_check?)) && (!Customer.require_risk_form? || signed_risk_form?) && (!Customer.require_conduct_form? || signed_conduct_form?)&& currently_enrolled? && (!Customer.mentor_workshop_event_type || attended_mentor_workshop?)
+    (!Customer.require_background_checks? || (passed_background_check? && passed_sex_offender_check?)) && (!Customer.require_risk_form? || signed_risk_form?) && (!Customer.require_conduct_form? || signed_conduct_form?)&& currently_enrolled? && (!Customer.mentor_workshop_event_type || attended_mentor_workshop?) && (!Customer.require_parental_consent_for_minors? || is_18?)
```

Comment:
I think this needs to be modified just a bit to first check `is_18?`, and if not, then check `!Customer.require_parental_consent_for_minors? || parental_consent of file`, because this will currently always fail for a mentor who is under 18 even if they have a form on file. It might be best to refactor all of those checks into a helper method that is called on this line.